bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/ladda/LaddaAjaxButton.java

```diff
@@ -16,7 +16,7 @@
  * that disables itself during the Ajax call and shows a loading spinner
  */
 public class LaddaAjaxButton extends BootstrapAjaxButton {
-
+    private static final long serialVersionUID = 1L;
```

Comment:
Same here.

Reply:
I see no issues with having default `serialVersionUID`
Will prefer to keep it :)